### DISCUSSION TITLE
Not functioning under windows

### DISCUSSION THREAD

**Discussion by candre23** [Girbons/comics-downloader#105] [Q&A]
I'm posting this here instead of under issues because it's probably a "me problem".

I've tried both the GUI and command line versions on win10.  The GUI version was crashing and closing itself after creating the folder structure, so I switched to the CLI version.  That worked properly for maybe half a dozen comics, but then stopped.  Now it crates the folders and CBR files, but they're only 22 bytes in size.  Weirdly, I went back and re-downloaded one of the comics that worked, and the 2nd time around it created these 22 byte files.

After the CLI version stopped working, I tried the GUI version again and it is now "working" insofar as it doesn't crash, but is also creating 22 byte CBRs.  I also tried one of the other supported sites, just in case I got blacklisted or something for scraping roughly 20 comics over the course of half an hour.  Same result.

Any suggestions?  I'm attaching a screenshot of the debug output and one of the 22 byte CBRs, just in case that's helpful

[Lemonade-Code-TPB-Part-1.zip](https://github.com/Girbons/comics-downloader/files/7777224/Lemonade-Code-TPB-Part-1.zip)

![log](https://user-images.githubusercontent.com/36796363/147413811-3d44df21-b2a4-4d90-907c-85ee15f5eeba.JPG)
.

**Comment by Girbons** [comment]
Hi @candre23, 

This issue is really similar to https://github.com/Girbons/comics-downloader/issues/88, where a captcha appears after a while. I don't have a workaround for this behaviour yet, as I'm barely maintaining this project.
